overlord/ifacestate/ifacemgr.go

```diff
@@ -62,6 +62,7 @@ func Manager(s *state.State, extra []interfaces.Interface) (*InterfaceManager, e
 	runner.AddHandler("setup-profiles", m.doSetupProfiles, m.doRemoveProfiles)
 	runner.AddHandler("remove-profiles", m.doRemoveProfiles, m.doSetupProfiles)
 	runner.AddHandler("discard-conns", m.doDiscardConns, m.undoDiscardConns)
+	runner.AddHandler("discard-namespace", m.doDiscardNamespace, nil)
```

Comment:
Should probably be done inside discard-snap. That's where we remove the snap files, and there's no undo there either ATM (might change with trashed data + cleanup), so if we want to undo a discard we need to fix both cases.